interview/algorithm/sort.py

```diff
@@ -0,0 +1,13 @@
+def quicksort(array):
+    """Sort array in ascending order
+
+    :param list array: unordered list of numbers/letters.
+    :return: (*list*) -- ordered list of numbers/letters.
+    """
+    if len(array) <= 1:
+        return array
+    else:
+        pivot = array[0]
+        lower = [x for x in array[1:] if x < pivot]
+        greater = [x for x in array[1:] if x > pivot]
```

Comment:
Should we make it in-place as classic quick sort has memory complexity O(1)?

Reply:
Good point

Reply:
Done